src/main/java/seedu/address/commons/util/IcsUtil.java

```diff
@@ -1,160 +1,358 @@
 package seedu.address.commons.util;
 
+import static biweekly.util.DayOfWeek.valueOfAbbr;
 import static java.util.Objects.requireNonNull;
 
+import java.io.File;
+import java.io.FileNotFoundException;
 import java.io.IOException;
-import java.nio.file.Files;
 import java.nio.file.Path;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
 import java.time.DayOfWeek;
 import java.time.LocalDate;
+import java.time.LocalDateTime;
 import java.time.LocalTime;
+import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
+import java.time.temporal.TemporalAdjusters;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
 import java.util.Optional;
 import java.util.logging.Logger;
 
+import biweekly.Biweekly;
+import biweekly.ICalendar;
+import biweekly.component.VEvent;
+import biweekly.property.DateEnd;
+import biweekly.property.DateStart;
+import biweekly.property.RecurrenceRule;
+import biweekly.property.Summary;
+import biweekly.util.Frequency;
+import biweekly.util.Recurrence;
+
 import seedu.address.commons.core.LogsCenter;
-import seedu.address.commons.exceptions.DataConversionException;
 import seedu.address.model.person.TimeSlot;
 import seedu.address.model.person.TimeTable;
-import seedu.address.model.person.exceptions.TimeSlotOverlapException;
 
 /**
- * Helps with reading from and writing to ICS files.
- * Only support ICS to-and-from TimeTable Objects.
- * TODO: create a 'serialisable' class to serialise(?) the timetable objects to ics file formats,
- * just like in the jsonUtil class
+ * Converts a TimeTable object instance to .ics and vice versa.
+ *
+ * Usage:
+ * 1) Mainly used to read and write the TimeTable(s) of FreeTime into the disk for permanent storage.
+ * 2) Also used during import and export commands (ie, read and write)
+ *
+ * NOTE: Only support ICS to-and-from TimeTable Objects(!!).
+ * TODO: allow conversion to objects other than timetable
+ * TODO: create a 'serialisable' class to serialise(?) the timetable objects to ics file formats
```

Comment:
I don't think "serializable" actually has any special meaning lol, so I don't think this is necessary